cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -197,6 +187,25 @@ pub enum Inst {
     /// Marker, no-op in generated code: SP "virtual offset" is adjusted. This
     /// controls how MemArg::NominalSPOffset args are lowered.
     VirtualSPOffsetAdj { offset: i64 },
+
+    /// mov between XMM registers (32 64) (reg addr) reg
+    /// XMM_Mov_RM_R differs from XMM_RM_R in that the dst
+    /// register of XMM_MOV_RM_R is not used in the computation
+    /// of the instruction dst value and so does not have to
+    /// be a previously valid value. This is characteristic of
+    /// mov instructions.
+    XMM_Mov_RM_R {
```

Comment:
nit: can you keep it in the sections of vcode insts labelled with a "FP instructions" header, please? I tried to order the vcode instructions in logical groups, and it would be nice if we tried to enforce it.

(also nit: code comments are allowed to use 99 chars per line, as in the rest of the code base, so the wrapping around can be reverted too)